packages/yarnpkg-core/sources/Project.ts

```diff
@@ -55,6 +55,19 @@ const FETCHER_CONCURRENCY = 32;
 const gzip = promisify(zlib.gzip);
 const gunzip = promisify(zlib.gunzip);
 
+export enum InstallMode {
+  /**
+   * Doesn't run the link step, and only fetches what's necessary to compute
+   * an updated lockfile.
+   */
+  LockfileUpdate = `lockfile-update`,
+
+  /**
+   * Don't run the build scripts.
+   */
+  SkippedBuilds = `skipped-builds`,
```

Comment:
What do you think of `skip-build` instead of `skipped-builds`?

Reply:
Fine by me, although then `lockfile-update` has to be renamed `update-lockfile` as well.